src/Altinn.Apps/AppTemplates/AspNet/App/logic/Validation/ValidationHandler.cs

```diff
@@ -1,17 +1,23 @@
-using System;
 using System.Threading.Tasks;
 using Altinn.Platform.Storage.Interface.Models;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc.ModelBinding;
-// using Altinn.App.Models; // Uncomment this line to refer to app model(s)
 
+//// using Altinn.App.Models; // Uncomment this line to refer to app model(s)
 
 namespace Altinn.App.AppLogic.Validation
 {
+    /// <summary>
+    /// Represents a business logic class responsible for running validation at different steps of a process.
+    /// </summary>
     public class ValidationHandler
     {
         private IHttpContextAccessor _httpContextAccessor;
 
+        /// <summary>
+        /// Initialize a new instance of the <see cref="ValidationHandler"/> class with access to the Http Context.
+        /// </summary>
+        /// <param name="httpContextAccessor">An http context accessor.</param>
         public ValidationHandler(IHttpContextAccessor httpContextAccessor = null)
```

Comment:
Once we're in this class making adjustments.. I don't like that we set a default value for the httpcontextaccessor.. Either we don't add it as input in the template or we send the context accessor as input from app.cs where the validationhandler is instantiated.. How it is currently implemented often causes a null refrence exception when the user tries to access the context, but isn't aware of the default assignment to null

Reply:
I agree, but I'm trying to avoid too many changes outside those that are needed to pass the StyleCop checks. 

I have made other changes outside that scope, but I don't want to take on too much as it dilutes the focus. We can create new issues if there are things we'd like to fix.